src/wp-includes/general-template.php

```diff
@@ -591,11 +591,11 @@ function bloginfo( $show = '' ) {
  *
  * - 'name' - Site title (set in Settings > General)
  * - 'description' - Site tagline (set in Settings > General)
- * - 'wpurl' - The WordPress address (URL) (set in Settings > General)
+ * - 'wpurl' - The ClassicPress address (URL) (set in Settings > General)
  * - 'url' - The Site address (URL) (set in Settings > General)
  * - 'admin_email' - Admin email (set in Settings > General)
  * - 'charset' - The "Encoding for pages and feeds"  (set in Settings > Reading)
- * - 'version' - The current WordPress version
+ * - 'version' - The current ClassicPress version
```

Comment:
This should say something like "A WordPress-style version number that can be used for feature comparisons".  We should also update the inline docs to this function if we make a change like https://github.com/ClassicPress/ClassicPress/pull/32#discussion_r215516465.